src/module_wrap.h

```diff
@@ -22,6 +22,11 @@ v8::Maybe<url::URL> Resolve(Environment* env,
                             const url::URL& base,
                             PackageMainCheck read_pkg_json = CheckMain);
 
+enum ScriptType {
```

Comment:
`enum class`

Reply:
won't this prevent me from using ScriptType without explicit coercion to int?